database/factories/GenreFactory.php

```diff
@@ -21,8 +21,9 @@ class GenreFactory extends Factory
      */
     public function definition()
     {
+
         return [
-            //
+
```

Comment:
For this, you can have this filled out too, something like:

See the comment I made on Genre Seeder about adding constants to the Model 😃 

```
return [
     'name' => Genre::GENRE_ACTION
        ]
```